bastionzero/environment/resource_environment_test.go

```diff
@@ -93,6 +93,60 @@ func TestAccEnvironment_Disappears(t *testing.T) {
 	})
 }
 
+func TestAccEnvironment_Name(t *testing.T) {
+	ctx := context.Background()
+	rDesc := acctest.RandomName()
```

Comment:
Remove checking description since description modification is already checked in test after this one

Reply:
[ec55904](https://github.com/bastionzero/terraform-provider-bastionzero/pull/35/commits/ec559047f3e63457fc457c3e04329f2b68a23d0e)